licenses/license.xml

```diff
@@ -0,0 +1,168 @@
+<?xml version='1.0'?>
+<licenseSummary>
+    <project>quickstart-backbone-app</project>
+    <version>0.0.0</version>
+    <license>Apache-2.0</license>
+    <dependencies>
+        <dependency>
+            <packageName>backbone</packageName>
+            <version>1.3.3</version>
+            <licenses>
+                <license>
+                    <name>MIT License</name>
+                    <url>http://www.opensource.org/licenses/MIT</url>
+                </license>
+            </licenses>
+        </dependency>
+        <dependency>
+            <packageName>fh-js-sdk</packageName>
+            <version>2.19.0</version>
+            <licenses>
+                <license>
+                    <name>UNKNOWN</name>
+                    <url>UNKNOWN</url>
```

Comment:
Isn't this Apache2? I believe all of the FH projects are under the Apache license? 

https://github.com/feedhenry/fh-js-sdk/blob/master/LICENSE